File: Javascript/Javascript Fundamentals/OperatorsAndExpressions/rectangles.js
function solve(args) {
	var width = +args[0];
    var height = +args[1];

	var perimeter = 2 * (width + height);
	var area = width * height;

	console.log(parseFloat(area).toFixed(2) + " " + parseFloat(perimeter).toFixed(2));
}

File: C#/C# Fundamentals/OperatorsAndExpressions/Rectangles/Rectangles.cs
/* 
   Write an expression that calculates rectangle’s area and perimeter by given width and height. 
   The width and height should be read from the console.
*/

using System;

class Rectangle
{
    static void Main()
    {
        double width = double.Parse(Console.ReadLine());
        double height = double.Parse(Console.ReadLine());

        double area = (2 * width) + (2 * height);
        double perimeter = width * height;

        Console.WriteLine("{0:F2} {1:F2}", perimeter, area);
    }
}